l10n_br_account_product/models/account_product_fiscal_classification.py

```diff
@@ -215,6 +220,38 @@ def _compute_taxes(self):
         ('account_fiscal_classfication_code_uniq', 'unique (code)',
          u'Já existe um classificação fiscal com esse código!')]
 
+    @api.multi
+    def get_ibpt(self):
+
+        brazil = self.env['res.country'].search([('code', '=', 'BR')])
+        states = self.env['res.country.state'].search([('country_id', '=',
+                                                        brazil.id)])
+        company = self.env.user.company_id
+        config = DeOlhoNoImposto(company.ipbt_token,
+                                 punctuation_rm(company.cnpj_cpf),
+                                 company.state_id.code)
+        tax_estimate = self.env['l10n_br_tax.estimate']
```

Comment:
Can you iterate over self? it is a multi method.
This way would be easy to add a context button in tree view, to execute over a bunch of models together.